Program.cs

```diff
@@ -3,17 +3,32 @@
 // Copyright (c) Metamation India.                                                
 // ---------------------------------------------------------------------------
 // Program.cs                                                                     
-// 
-// 
-// 
+// Program to print the frequency of each charcter in the words.txt file in descending order.
+// Also displays the most popular seeds for the spelling bee game.
 // ---------------------------------------------------------------------------------------
-using static System.Console;
+
 namespace Training {
    #region class Program ------------------------------------------------------------------------
+   /// <summary>Spelling Bee Frequency</summary>
    internal class Program {
       #region Method ----------------------------------------------
-      static void Main (string[] args) {
-         Console.WriteLine ("Hello, World!");
+      /// <summary>This method calls the SpellBeeFreq method</summary>
+      static void Main () => SpellBeeFreq ();
+
+      /// <summary>This method reads all lines from the text file 
+      /// and displays the frequency of each character in descending order 
+      /// and the most popular seeds for the spell bee game</summary>
+      static void SpellBeeFreq () {
+         // Frequency of each character in the text file in descending order
+         string words = String.Join ("", File.ReadAllLines ("C:/etc/words.txt"));
+         var freqList = words.GroupBy (x => x)
+                         .OrderByDescending (x => x.Count ())
+                         .ToList ();
+         foreach (var l in freqList) Console.WriteLine ($"{l.Key}  -  {l.Count ()} time(s)");
+
+         // Most frequent seeds for spell bee game
+         var seed = freqList.Take (7).Select (x => x.Key).ToArray ();

```

Comment:
Why do you need to convert the sequence to an array? The sequence can directly be used in the line below:
<img width="826" alt="image" src="https://github.com/RaaneshKumar/Training/assets/75829870/d282c8ec-1657-4434-939f-a78a65f78f7c">